equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/CrmMapper.java

```diff
@@ -43,6 +44,13 @@ default String convertDecisionReason(CrmDecisionReasonDetailsModelInterface mode
                 model.getFull_grant_notes() : model.getReason_details();
     }
 
+    @Nullable
+    default Boolean convertIntegerToBoolean(Integer i) {
```

Comment:
Do we need to mention the `default` keyword here?

Reply:
yes. it is the only way to declare a default function on an interface